_posts/2018-10-23-cryptape.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "This Week in RustFest 5: Cryptape at RustFest Rome 2018!"
```

Comment:
Hm. At least in the past we never used "TWIRF" in the title for these blog posts, since they are fully sponsored and don't have anything to do with us (also note these posts are also different from the "mention in TWIRF" perk, they are the "blog post before event" perk).

Reply:
make perfectly sense! I didn't copy correctly the template.

thanks!